doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -419,11 +419,13 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
    ACRN supports three channels: ``ioc``, ``power button`` and ``uart``.
```

Comment:
I think the correct parameter is ``power_button`` (with a underscore and not a space)